src/app/Http/Controllers/AdminController.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Backpack\CRUD\app\Http\Controllers;
 
+use Facade\FlareClient\Flare;
```

Comment:
This just hit me. Doesn't this mean that we'll need to add Flare as a dependency to Backpack? AFAIK it does ship with Laravel starting from a certain version, but... there's a way around it, and some people use Whoops instead. What will happen for those people?

Also, how often do Flare push breaking changes? Fuck 😅  The more I think about this, the more I think loading a Flare error is more consistent (sure), but not worth it.

Reply:
No more Flare 🙌
It's a custom error frame.